### ISSUE TITLE
Font-Size: 1.03rem

### ISSUE THREAD

**Issue by caillou** [xz/new.css#12]
The `body` has a confusing `font-size` of `1.03rem`.

Given that most browsers have a base font-size of `16px`, we end up with a `font-size` of `16.43px`. This seems like an odd choice, especially on normal-density screens.

What is the rational behind this value? 
If this has a reason, could you add a comment?

**Comment by halfmage** [comment]
I thought about the same. This affects not only the body text to have `16.43px` but also all other elements that have rem values for there size, padding, and margin. Fixed it in PR #25

**Comment by caillou** [comment]
Actually it does not affect the `rem` as the `font-size` is set on the body. AFIK the `rem` unit is based on the `font-size` defined on the the `html` element.

> rem values were invented in order to sidestep the compounding problem. rem values are relative to the root html element, not the parent element.

– [mdn/font-size](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size)

**Comment by halfmage** [comment]
Yes you are right. My point was caused by stuff like `.9rem` in one case. But anyway I would prefer 1rem for body text size.

**Comment by 3x** [comment]
Will take a look at these. Thanks!
